catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -203,6 +222,40 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
         outer.validators = validators;
     }
 
+    @SuppressWarnings("unchecked")
+    private void parseMetacardValidators(@Nullable Map<String, Object> root, Outer outer) {
+        if (root == null || root.get(METACARD_VALIDATORS_PROPERTY) == null) {
+            return;
+        }
+
+        List<MetacardValidatorDefinition> metacardValidators = new ArrayList<>();
+        Object metacardValidatorsObj = root.get(METACARD_VALIDATORS_PROPERTY);
+        if (metacardValidatorsObj instanceof List) {
+            List<Map<String, Object>> metacardValidatorsList =
+                    (List<Map<String, Object>>) metacardValidatorsObj;
+            for (Map<String, Object> metacardTypeList : metacardValidatorsList) {
+                for (Map.Entry<String, Object> typeEntry : metacardTypeList.entrySet()) {
+                    String metacardType = typeEntry.getKey();
+                    Object validatorDefinitionObj = typeEntry.getValue();
+                    if (validatorDefinitionObj instanceof List) {
+                        List<Map<String, Object>> validatorEntryList = (List<Map<String, Object>>) validatorDefinitionObj;
+                        for (Map<String, Object> metacardDefinitionMap : validatorEntryList) {
+                            MetacardValidatorDefinition validatorDefinition = new MetacardValidatorDefinition();
+                            validatorDefinition.metacardType = metacardType;
+                            Map<String, Object> arguments = new HashMap<>(2);
```

Comment:
Changing this to directly pass in the Map as a copy instead of doing the copy myself. This code changed a little and that is no longer necessary.